features/polyExplorer/src/components/dataViz/purposesBarChart.jsx

```diff
@@ -14,10 +14,10 @@ const PurposesBarChart = ({ data, animation, onClick }) => {
 
     const svgBarRef = useRef();
     const gHeight = 52;
-    const width = 500;
+    const width = 400;
     const height = data.length * (gHeight * 2);
     const margin = { top: 36, right: 30, bottom: 1, left: 1 };
-    const barHeight = gHeight - margin.top;
+    const barHeight = 16;
```

Comment:
Still the same value, but fixed?

Reply:
I changed it as a fixed number because the bars must be that size always and, in case we need to change the margin-top or `gHeight`, which are more related to the position, the  bar size remains the same.

Reply:
If that's *always* the case, maybe move it to poly-look?